Makefile

```diff
@@ -0,0 +1,19 @@
+INIT_TASKS:=		create-precommit dependency-install
```

Comment:
I think we should vote the use of `Makefile` probably is a bit out of scope for the actual issue this PR solves.

Reply:
What are the cons to not use it?

Reply:
Is just not useful, npm scripts are able to perform the same.